.github/workflows/main.yml

```diff
@@ -86,3 +86,35 @@ jobs:
               name: ${{ env.playwright-upload-result-path }}
               path: ${{ steps.test-result.outputs.result-path }}
               retention-days: 1
+
+   record-playwright-metrics:
+      name: record-playwright-metrics
+      runs-on: [self-hosted, metrics-runner]
+      needs: [playwright-run]
+      if: success() || failure()
+      timeout-minutes: 5
+      steps:
+         - name: Checkout ifixit
+           uses: actions/checkout@v3
+           with:
+              repository: iFixit/ifixit
+              token: ${{ secrets.ACTION_PAT }}
+
+         - name: Download playwright test results
+           if: always()
```

Comment:
Why do we need the always here if we already have it on the entire job?

Reply:
Oops, we can drop this. I was using this for debugging.